index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="style.css">
```

Comment:
Great work adding your CSS to the page. Instead of doing all of your CSS in one file, however, you should conform to the SMACSS principles and break out your CSS into a few files (base, module, layout, etc.), even if you don't have much CSS yet.